cropduster/templatetags/cropduster_tags.py

```diff
@@ -42,29 +42,27 @@ def get_crop(image, crop_name, size=None, attribution=None, exact_size=False):
     values so `exact_size` gives you access to the actual size of an image.
     """
 
-    if not image:
+    if not image or not image.related_object:
         return
 
     if size:
         warnings.warn("The size kwarg is deprecated.", DeprecationWarning)
 
+    if exact_size:
+        warnings.warn("The exact_size kwarg is deprecated.", DeprecationWarning)
+
```

Comment:
Well, it predates a lot of cropduster/generic-pluses functionality (prefetch etc). It makes sense than we'd refactor it to work with the updates.

The hard part about moving it onto the field is we'd have to get rid of crops with `-`'s in the names, or automatically infer they're the same as `_`, since we could do something like `article.lead_image.lead_large` in the template, but not `article.lead_image.lead-large`.